Scripts/whitelist_blacklist.sqf

```diff
@@ -101,6 +101,7 @@ if(isNil "bluforWhiteList") then {
 		"76561198119163382", //Chris Kyle
 		"76561198146926838", //WakeUpEatPlaySleep
 		"76561198211441867" //Kevlar
```

Comment:
Needs a coma after "76561198211441867" should be

`"76561198211441867", //Kevlar`